kiwi/builder/disk.py

```diff
@@ -479,12 +482,18 @@ def create_disk(self) -> Result:
         if self.system_setup.script_exists(
             defaults.POST_DISK_SYNC_SCRIPT
         ):
+            image_system = ImageSystem(
+                device_map, self.root_dir,
+                system.get_volumes() if self.volume_manager_name else {}
+            )
+            image_system.mount()
```

Comment:
I see this mounts the root device. I am wondering where it was mounted before this change.

Reply:
yeah this was one of the reasons why it's now imho nicer. Before this change it was this part from the diff

```
             disk_system = SystemSetup(
-                self.xml_state, system.get_mountpoint()
+                self.xml_state, image_system.mountpoint()
             )
```

so the mountpoint was only ```system.get_mountpoint()``` which only points to what the root partition is and not more. At call time this partition was mounted due to the sync action which is encoded in the ```_sync_system_to_image``` method that runs before